### PR TITLE
Spawner

### PR DESCRIPTION
**Pull request by p33rs**
Refactors Epilepsy and Storm.
Adds Cursor Improvement Engine for the hell of it.

### CODE REVIEW

**Comment by nicktacular** [comment]
After careful consideration, I've decided this is a must-have on all websites. I am telling all my :horse: friends.

File: dist/horse.js [hunk 1]
```diff
@@ -17,37 +96,15 @@ Epilepsy.prototype.defaultOptions = {
...
-    return this;
-};
-
 Epilepsy.prototype.validateOptions = function(options) {
     return (
         options &&
-            (options.spawners === 'number' && options.spawners > 0 ) &&
-            (options.minDelay === 'number' && options.minDelay >= 0 ) &&
-            (options.maxDelay === 'number' && options.maxDelay >= 0 ) &&
+            (typeof options.spawners === 'number' && options.spawners > 0 ) &&
```
Comment (deforestg) [inline dist/horse.js:102]: good god such logic chain.. it should just have a default setup that simply works no matter what crap gets passed. if you wanna validate so much stuff you need a better way, this is :hankey:

Comment (nicktacular) [inline dist/horse.js:102]: No, this is beautiful. Such logic. :horse: would be proud :hankey:

Comment (p33rs) [inline dist/horse.js:102]: :horse: is not an excuse for :horse_racing: 

all :poop: should be :cake: even if it is :horse: :poop:

**Pull request merged**